src/Illuminate/Collections/Arr.php

```diff
@@ -23,6 +28,21 @@ public static function accessible($value)
         return is_array($value) || $value instanceof ArrayAccess;
     }
 
+    /**
+     * Determine whether the given value is arrayable.
+     *
+     * @param  mixed  $value
+     * @return bool
+     */
+    public static function arrayable($value)
```

Comment:
Wouldn't it make sense to use a prefix here to indicate the return value? Because, I would expect the method to convert it from array-like values to `Arrayable` instances
```suggestion
    public static function isArrayable($value)
```

Reply:
I followed naming convention used in `Arr::accessible`.

Reply:
Ah, thanks for the explanation 👍🏻